packages/elements/src/interactive-chart/index.ts

```diff
@@ -1,4 +1,4 @@
-import { ColorType, createChart as chart } from 'lightweight-charts';
+import { ColorType, Time, createChart as chart } from 'lightweight-charts';
```

Comment:
```suggestion
import { ColorType, createChart as chart } from 'lightweight-charts';
```
I saw `import type` at line number 33. We should do at that.